showcase/src/app/app.component.html

```diff
@@ -34,6 +34,9 @@
 			<a mat-list-item uiSref="demo-button" uiSrefActive="active">
 				<span matLine>Button</span>
 			</a>
+			<a mat-list-item uiSref="demo-date-picker" uiSrefActive="active">
+					<span matLine>Date Picker</span>
```

Comment:
There is an alignment issue. Could you please solve it ?

Reply:
Indeed, I did run prettier-check though...